src/main/resources/static/_shared.yml

```diff
@@ -4114,3 +4114,67 @@ components:
         - A1
         - A2
         - A3
+    NewAppeal:
+      type: object
+      properties:
+        appealDate:
+          type: string
+          format: date
+        appealDetail:
+          type: string
+        reviewer:
+          type: string
+      required:
+        - appealDate
+        - appealDetail
+        - reviewer
```

Comment:
As discussed, I think we want to create appeals endpoint to be a one-shot endpoint (at least for now). This makes the UI work easier to implement

Reply:
This has now been changed to not require a separate endpoint for adding the decision.